SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMigration/SpatialTestCharacterMigration.cpp

```diff
@@ -38,19 +38,29 @@ ASpatialTestCharacterMigration::ASpatialTestCharacterMigration()
 	Author = "Victoria";
 	Description = TEXT("Test Character Migration");
 	TimeLimit = 300;
+
+	Destination = FVector(200.0f, 0.0f, 40.0f);
```

Comment:
I'd rather these were set from `UAbstractLBStrategy::GetWorkerEntityPosition`

Reply:
Or at least some validation that Destination and Origin are on different servers and handover is taking place. You could monitor authgained/lost calls, or use `WhoShouldHaveAuthority` perhaps

Reply:
Think GetWorkerEntityPosition does not actually tell you where the boundary is? With this 2 zone grid, I get a "WorkerEntityPosition " of { 250000, 0, 0 }.

Reply:
nit: might be better to rename these - PositionOnServerOne etc.